engine/index.md

```diff
@@ -2,62 +2,129 @@
-* APIs which specify interfaces that programs can use to talk to and
-  instruct the Docker daemon.
-* A command line interface (CLI) client [`docker`](/engine/reference/commandline/cli/).
+- A server with a long-running daemon process
+  [`dockerd`](/engine/reference/commandline/dockerd).
+- APIs which specify interfaces that programs can use to talk to and instruct
+  the Docker daemon.
+- A command line interface (CLI) client
+  [`docker`](/engine/reference/commandline/cli/).
 
 The CLI uses [Docker APIs](api/index.md) to control or interact with the Docker
 daemon through scripting or direct CLI commands. Many other Docker applications
 use the underlying API and CLI. The daemon creates and manage Docker objects,
 such as images, containers, networks, and volumes.
 
-For more details, see [Docker Architecture](../get-started/overview.md#docker-architecture).
-
-## Docker user guide
-
-To learn about Docker in more detail and to answer questions about usage and
-implementation, check out the [overview page in "get started"](../get-started/overview.md).
-
-## Installation guides
-
-The [installation section](install/index.md) shows you how to install Docker
-on a variety of platforms.
-
-## Release notes
-
-A summary of the changes in each release in the current series can now be found
-on the separate [Release Notes page](release-notes/index.md)
-
-## Feature Deprecation Policy
-
-As changes are made to Docker there may be times when existing features
-need to be removed or replaced with newer features. Before an existing
-feature is removed it is labeled as "deprecated" within the documentation
-and remains in Docker for at least 3 stable releases unless specified
-explicitly otherwise. After that time it may be removed.
-
-Users are expected to take note of the list of deprecated features each
-release and plan their migration away from those features, and (if applicable)
-towards the replacement features as soon as possible.
-
-The complete list of deprecated features can be found on the
-[Deprecated Features page](deprecated.md).
+For more details, see
+[Docker Architecture](../get-started/overview.md#docker-architecture).
+
+<div class="component-container">
+  <!--start row-->
+  <div class="row">
+    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+      <div class="component">
+        <div class="component-icon">
+          <a href="/engine/install/"><img src="/assets/images/download.svg" alt="Arrow pointing downwards" width="70px" height="70px"></a>
+        </div>
+        <h2><a href="/engine/install/">Install Docker Engine</a></h2>
+        <p>Learn how to install the open source Docker Engine for your distribution.</p>
+      </div>
+    </div>
+    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-4 block">
+      <div class="component">
+        <div class="component-icon">
+          <a href="/storage/"><img src="/assets/images/engine-storage.svg" alt="Data disks" width="70px" height="70px"></a>
```

Comment:
yeah I was thinking about that, but I think a prerequisite to that would be that we actually do an inventory of our iconography. It's currently a mix of icons from bootstrap, fontawesome, material design/dd, etc. In the meantime I made sure to namespace these as to avoid any confusion...